src/proto/proxy.proto

```diff
@@ -16,6 +16,7 @@ message NetworkConfig {
   string host_name = 5;
   Version version = 6;
   string ns_path = 7;
+  string container_id = 8;
```

Comment:
I have not tested this as we never made any changes to the over the wire format but one thing that MUST be considered is that the dhcp proxy is a long running service. This means a new client sending a new format must still work with the old service running. So it would be important to know if the new field would cause the old service to fail.

Reply:
Ah, I hadn't thought of that. Easily tested though.

Reply:
Works fine...
* Stopped all containers and let dhcp-proxy stop after 30 seconds.
* Reinstalled podman and netavark from Fedora packages.
* Created and started a new container with macvlan/dhcp.
* No issues.  As expected, container did not get hostname as container name and neither client id nor hostname were sent to DHCP server.
* Confirmed dhcp-proxy was running.
* Installed netavark built from PR.
* Verified that the dhcp-proxy was still running with the original PID.
* Created and started a new container with macvlan/dhcp.
* No issues.  Again, as expected, container did not get hostname as container name and neither client id nor hostname were sent to DHCP server.
* Installed podman built from my PR with containers still running.
* Created and started another new container with macvlan/dhcp.
* No issues.  As expected, container now got hostname as container name and neither client id nor hostname were sent to DHCP server.
* Stopped all containers and let dhcp-proxy stop itself after 30 seconds.
* Restarted the previous container.
* As expected, container got hostname as container name and both client id and hostname were sent to DHCP server.